.github/workflows/build-backend-function.yml

```diff
@@ -0,0 +1,63 @@
+# This workflow will build a backend cloud function with Gradle
+# For more information see: https://github.com/google-github-actions/deploy-cloud-functions
+
+name: Build and deploy backend cloud function
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build-and-deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up AdoptOpenJDK 11
+        uses: joschi/setup-jdk@v2
+        with:
+          java-version: 11
+
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle') }}
+          restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      # Set up ADC for credentials
+      # Ideally this step should be part of action.yml but
+      # GHA does not yet support action within action so we're stuck.
+      # Please refer to https://github.com/actions/runner/pull/612 for updates.
+      - uses: google-github-actions/setup-gcloud@master
+        with:
+          service_account_email: ${{ secrets.DEPLOY_CF_SA_EMAIL }}
+          service_account_key: ${{ secrets.DEPLOY_CF_SA_KEY_JSON }}
```

Comment:
have all the secrets stored in this repo been vetted/approved by devops and/or appsec?

Reply:
There are no formal approval yet by `devops` or `appsec` on the compliance/security-level of these keys. These secrets are for GCP `terra-kernel-k8s` only.

Reply:
The creation on the BQ tables in Test Runner is different from creation of snapshots in `Delta Layer`.  Let's have appsec weighs in and see what's recommended to meet their standards.